.github/workflows/ci.yaml

```diff
@@ -215,21 +215,7 @@ jobs:
           poetry install
 
       - name: Install bitcoind
-        run: |
-          (
-            cd /tmp/
-            wget https://storage.googleapis.com/c-lightning-tests/bitcoin-$BITCOIN_VERSION-x86_64-linux-gnu.tar.bz2
-            wget -q https://storage.googleapis.com/c-lightning-tests/elements-$ELEMENTS_VERSION-x86_64-linux-gnu.tar.bz2
-            tar -xjf bitcoin-$BITCOIN_VERSION-x86_64-linux-gnu.tar.bz2
-            tar -xjf elements-$ELEMENTS_VERSION-x86_64-linux-gnu.tar.bz2
-            sudo mv bitcoin-$BITCOIN_VERSION/bin/* /usr/local/bin
-            sudo mv elements-$ELEMENTS_VERSION/bin/* /usr/local/bin
-            rm -rf \
-              bitcoin-$BITCOIN_VERSION-x86_64-linux-gnu.tar.gz \
-              bitcoin-$BITCOIN_VERSION \
-              elements-$ELEMENTS_VERSION-x86_64-linux-gnu.tar.bz2 \
-              elements-$ELEMENTS_VERSION
-          )
+        run: bash -x .github/scripts/install-bitcoind.sh
```

Comment:
If you're invoking it this way, you don't need the shebang line in the script.  But use `sh ....` instead of `bash -x ....`?